dodo.py

```diff
@@ -12,8 +14,22 @@ def integration_test_action():
             'actions': [action]
         }
 
+def task_graphviz():
+    graph_dir = "docs\\graphs\\"
```

Comment:
The `\` as path separator only works on Windows, my machine does nothing when I `doit graphviz`.

I guess a more portable solution would be to set `graph_dir = ['docs', 'graphs']` and then call

```
os.path.join(*graph_dir, ...)
```

below in lines 19 and 25.

Reply:
Great, your solution is better than my proposal